common/supplemental/attributeValueValidity.xml

```diff
@@ -267,18 +267,18 @@
 			<attributeValues dtds='supplementalData' elements='approvalRequirement' attributes='locales' type='list'>$locale|$coverageSpecial</attributeValues>
 			<attributeValues dtds='supplementalData' elements='approvalRequirement' attributes='paths' type='TODO'/>
 			<attributeValues dtds='supplementalData' elements='approvalRequirement' attributes='votes' type='choice'>20 4 8</attributeValues>
-			<attributeValues dtds='supplementalData' elements='attributeValues' attributes='attributes' type='choice'>_q aliases allowed alt before
-				before from to contains type count date day digits hex id iso3166 iso4217 key multizone number numberSystem path preferred regions registry
-				replacement replacement type request rounding scripts source territories territory time type uri validSubLocales value variants version</attributeValues>
+			<attributeValues dtds='supplementalData' elements='attributeValues' attributes='attributes' type='choice'>_q aliases allowed alt before contains
+				count date day digits from hex id iso3166 iso4217 key multizone number numberSystem path preferred regions registry replacement request
+				rounding scripts source territories territory time to type uri validSubLocales value variants version</attributeValues>
```

Comment:
do we have a tool for this, or did it have to be done by hand (text editor)?

Reply:
In this case done by hand. I cloned the original ticket to https://unicode-org.atlassian.net/browse/CLDR-15881 to consider having CLDRModify do the sorting and duplicate removal.